src/DotLiquid/Strainer.cs

```diff
@@ -15,6 +15,27 @@ static class DictionaryExtensions
                 return dic[key] = factory();
             return found;
         }
+
+    }
+
+    static class MethodInfoExtensions
+    {
+        public static int GetNonContextParameterCount(this MethodInfo method)
+        {
+            return method.GetParameters().Count(p => p.ParameterType != typeof(Context));
+        }
+
+        public static bool MatchesMethod(this MethodInfo method, KeyValuePair<string, IList<Tuple<object, MethodInfo>>> compareMethod)
+        {
+            string methodName = Template.NamingConvention.GetMemberName(method.Name);

```

Comment:
inline single use parameters

Reply:
I fixed this here: https://github.com/dotliquid/dotliquid/pull/453/commits/72a2410fe218c54f5da077e7012d54d35c87b34c . I failed to make it's own commit.